velox/exec/Window.h

```diff
@@ -65,8 +75,8 @@ class Window : public Operator {
     const core::WindowNode::WindowType type;
     const core::WindowNode::BoundType startType;
     const core::WindowNode::BoundType endType;
-    const std::optional<column_index_t> startChannel;
-    const std::optional<column_index_t> endChannel;
+    std::optional<FrameChannelArg> startFrameArg;
```

Comment:
Why did you remove the const qualifier here ?

Reply:
Thanks for pointing it out, it was removed because an empty FrameArg object was being set from updateKBasedFrameBounds before, reintroduced const.